core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -1095,8 +1111,11 @@ public int compare(FingerprintFacet o1, FingerprintFacet o2) {
 
     /**
      * Finds a facet of the specific type (including subtypes.)
+     * @param <T> Class of the {@link FingerprintFacet}
+     * @return First matching facet of the specified class
```

Comment:
Seems to be "some" not "first", which is fine when you are only assuming one instance per class.